ports/raspberrypi/boards/bradanlanestudio_explorer_rp2040/pins.c

```diff
@@ -38,7 +38,7 @@ static const mp_rom_map_elem_t board_module_globals_table[] = {
 
     { MP_ROM_QSTR(MP_QSTR_GP0), MP_ROM_PTR(&pin_GPIO0) },
     { MP_ROM_QSTR(MP_QSTR_GP1), MP_ROM_PTR(&pin_GPIO1) },
-    // GPIO0 and GPIO1 are also the UART
+    // GPIO0 and GPIO1 are used for IR and not for  UART
```

Comment:
Did you want to change the names if they are not used for UART? (e.g. `IR_TX` and `IR_RX`, or anything like that?)